model/clusters_mgmt/v1/machine_type_type.model

```diff
@@ -42,4 +42,7 @@ class MachineType {
 
 	// 'true' if the instance type is supported only for CCS clusters, 'false' otherwise.
 	CCSOnly Boolean
+
+	// The machine root volume capabilities.
+	RootVolume AWSVolume
```

Comment:
Is this the correct option we want to go with? What about other providers? 

Would it make sense to have a RootVolume struct and nest under that struct the provider specific volumes. 

```
type RootVolume struct {
    AWS AWSVolume
    GCP GCPVolume
    ....
}
```

What do you think @vkareh @etabak

Reply:
SGTM